helpers/config.py

```diff
@@ -380,14 +380,14 @@ def get_template(cls):
             'postgres_replication_password': Config.generate_password(),
             'postgres_settings': False,
             'postgres_settings_content': '\n'.join([
-                '# Generated by PGConfig 3.0.0 (3af8ea764b2fd9ea8401acfdc69f90cb825cdee9)',
-                '# https://api.pgconfig.org/v1/tuning/get-config?environment_name=Desktop&format=conf&include_pgbadger=false&cpus=1&max_connections=100&pg_version=14&total_ram=2GB&drive_type=HDD',
+                '# Generated by PGConfig 3.1.0 (1d600ea0d1d79f13dd7ed686f9e2befc1fcf9226)',
+                '# https://api.pgconfig.org/v1/tuning/get-config?format=conf&log_format=csvlog&max_connections=100&pg_version=14&environment_name=Mixed&total_ram=2GB&cpus=1&drive_type=SSD&arch=x86-64&os_type=linux',
```

Comment:
Can we remove `log_format=csvlog` since it doesn't seem to do anything when `include_pgbadger=false`, which seems to be the default now? I also wonder if we shouldn't keep the explicit `include_pgbadger=false` just to be sure

Reply:
Right, I also removed `arch=x86-64` since it's the default and it's not used [here](https://github.com/kobotoolbox/kobo-install/blob/766b1800e0e119c345d31cc0ca0cf06e8f4d5cfc/helpers/config.py#L1661-L1671)